app/jobs/fraud_rejection_daily_job.rb

```diff
@@ -3,13 +3,20 @@ class FraudRejectionDailyJob < ApplicationJob
 
   def perform(_date)
     profiles_eligible_for_fraud_rejection.find_each do |profile|
-      analytics.automatic_fraud_rejection(verified_at: profile.verified_at)
+      analytics.automatic_fraud_rejection(
+        rejection_date: Time.zone.today,
+        verified_at: profile.verified_at,
+      )
       profile.reject_for_fraud(notify_user: false)
     end
   end
 
   private
 
+  def analytics(user: AnonymousUser.new)
+    Analytics.new(user: user, request: nil, session: {}, sp: nil)
```

Comment:
Do we want to add this method to ApplicationJob instead of here, to mirror the [definition in ApplicationController](https://github.com/18F/identity-idp/blob/19c0af192faea575a6549cc15e394cc0a3d98032/app/controllers/application_controller.rb#L59-L69)? It's memoized there and I wonder if we want to do that here too.

Where was `analytics` defined before this PR?

Reply:
It is defined explicitly in some other jobs like this. For example: [here ](https://github.com/18F/identity-idp/blob/main/app/jobs/in_person/email_reminder_job.rb#L29-L31). We could place it in ApplicationJob, but I'm wondering if that should be a separate story to move all calls to analytics in jobs to ApplicationJob.

Reply:
Making that a separate ticket makes sense to me.